### ISSUE TITLE
Design Intent  - IA Feedback - VA Integration - Ask VA

### ISSUE THREAD

**Issue by erinrwhite** [department-of-veterans-affairs/va.gov-team#62882]
## VFS actions
- [x] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

**Note:** @erinrwhite met with @sterkenburgsara who reviewed the recording and offered the lion's share of thoughts/questions and feedback, most of which is directly pasted here. Two IA's on this ticket!

- Thank you for setting out the scope of this project so well, helpful to know that backend logic updates are out of scope here.
- Really appreciate the proposed category names you all included in your artifacts to show us how you’ll update the drop-down options for plain language. "This is fantastic & I have no notes." - @sterkenburgsara 
- Echoing [feedback from @kristinoletmuskat](https://app.zenhub.com/workspaces/collaboration-cycle-reviews-61d89e9e5f4c120011a0ad37/issues/gh/department-of-veterans-affairs/va.gov-team/60696#issuecomment-1648708845) in #60696 re: entry points. It appears this will live at the root level on [VA.gov](http://va.gov/) - will there be any additional entry points beyond the existing link from the [Contact Us page](https://www.va.gov/contact-us/)?
- We're only seeing unauth wireframes here, which is fine for design intent - but curious if you will you use prefill to avoid some of these questions if users login? We'll want to see that version at midpoint.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Should**: Echoing @kristinoletmuskat's feedback, continue to work with IA folks from Sitewide CAIA for feedback and enhancement of user flows and entry points. 
- **Should:** Search with type-ahead results - yes do this whenever possible, try to align with similar search behavior used by other tools on [VA.gov](http://va.gov/) (Examples include: facilities search, GI bill comparison tool search)
- **Should:** try not to use drop-down menus if you can bucket this in fewer categories - the page may be more scannable for users if those options are visible as radio buttons (I can't see the full menu - so ignore this feedback if you have more than 10 items to manage) 
- **Consider:** Suggestion to pivot user flow: the primary reason users come to this tool is to ask a question. Putting that last in the flow distances users from what they are trying to do. Consider moving the actual text-box for the question up in the flow. 
- **Consider**: Your subheading description of Ask VA is really useful - I'd suggest adding that text under the page title on all pages of the product. Especially since the actual question field is so deep in the process.

**Comment by tygindraux** [comment]
Hi Erin and Sara! Thanks for your feedback. Here's our initial replies/work we've done so far:

- we've just started working with Kristin (from CAIA) on entry and exit points [here in Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1689629683164/ed237e8c59a4c0c9fb4ef9ced7e923297d7a3af6?wid=0-1690926738664&sender=u397f6dbbaed1b5b67b956658)
- we didn't show this at design intent, but we're also working on an authenticated flow [here in Sketch](https://sketch.com/s/7e494761-be10-4aef-a268-a45a02fbc5c3)
- we've started thinking about prefill options [here in a Google Doc](https://docs.google.com/document/d/1KKqut2SdP9ccgtUjlPDvlKuMm_fJqoHFkkgjSOD-V3U/edit?usp=sharing), and we'll aim to share more about this at midpoint review
- we've replaced drop down menus with radio buttons when there's less than 10 options
- we've moved the question earlier in the flow (we're also stoked about this! it was placed later in the flow due to conditionals/constraints, but since we moved to a one-question-per-page format we're not facing that constraint anymore!)
- we added the subheading to each page

We still need to:

- explore using search with type-ahead results (we're looking at the facilities locator as a reference)

I'll let you know/update this ticket once that's done. Thanks again.
